### DISCUSSION TITLE
Does Hobby plan make the source code public?

### DISCUSSION THREAD

**Discussion by qnxdev** [vercel/vercel#4331] [General]
I heard Vercel Hobby plan makes source of a project deployed on it public and anyone can see.

**Comment by paulogdm** [comment]
Negative. All hobby deployments should be private by default unless you are using the legacy platform.
You can check this by accessing `deploy.now.sh/_src` or `deploy.now.sh/_logs` in a new incognito window.

**Reply by qnxdev** [reply]
Thanks for the info. Maybe they were referring to the legacy version.
Both the links gave 404 in incognito.

**Reply by qnxdev** [reply]
How can I invest into Vercel?
